sklearn/metrics/_plot/base.py

```diff
@@ -38,3 +43,72 @@ def _check_classifier_response_method(estimator, response_method):
                 estimator.__class__.__name__))
 
     return prediction_method
+
+
+def _get_target_scores(X, estimator, response_method, pos_label=None):
```

Comment:
Using the word "scores" here may get confused with metrics and scorers. Maybe `_get_response` is more clear?